gen/generic_gen

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use perunServicesInit;
+use Storable;
+
+local $::PROTOCOL_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHierarchicalData;
+
+my $file = "/tmp/$::SERVICE_NAME-data";
```

Comment:
I would prefer to chose file name or just print it to stdout and let user to decide where he wants it.